lib/components/Alert.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react'
+import classNames from 'classnames'
+
+export interface Props {
+  type: 'success' | 'danger'
+  messages?: string | string[]
+}
+
+const Alert: FC<Props> = ({ type, messages }) => {
+  return messages ? (
+    <div className={classNames(['alert', `alert-${type}`])}>
+      {typeof messages === 'string' ? (
+        messages
+      ) : (
+        <ul>
+          {messages.map(message => (
+            <li key={message}>{message}</li>
+          ))}
+        </ul>
+      )}
+    </div>
+  ) : null
+}
```

Comment:
An empty success alert is rendered now.
It seems that `<Alert />` should return `null` if messages is empty array.

<img width="674" alt="Screen Shot 2020-04-27 at 23 48 29" src="https://user-images.githubusercontent.com/38908416/80385906-a4ac5d00-88e1-11ea-9dcf-89cd00bbde4f.png">

Reply:
Thanks, I fixed it.